app/src/main/res/values/strings.xml

```diff
@@ -250,4 +250,6 @@
 <string name="title_identity_management">SQRL - Identity Management</string>
 <string name="language_chinese">Chinese</string>
 <string name="action_password_options">Password options</string>
+<string name="idle_timeout_guidance">So you\'ll be able to remind yourself periodically of your SQRL master password, we only let you go one week (%1$s minutes), then you\'ll have to type it again.</string>
```

Comment:
I suggest to avoid words like "we", which could raise questions with our users (like *who exactly is "we"?*), and go for a more neutral form like for example

`So you don't forget your SQRL master password, the maximum QuickPass timeout is one week ( %1$s minutes), after which you\'ll have to type in the master password again.`

Reply:
I like yours better.  Mine sounds like I'm typing an email to my friend, hehe!